tools/code-generation/generator/src/main/java/com/amazonaws/util/awsclientgenerator/generators/cpp/CppClientGenerator.java

```diff
@@ -208,6 +208,37 @@ protected void addRequestIdToResults(final ServiceModel serviceModel) {
                 });
     }
 
+    private static Set<String> servicesMissingMultiAuthMRAPTrait = new HashSet<>();
```

Comment:
nit: could just do
```java
private static Set<String> servicesMissingMultiAuthMRAPTrait = ImmutableSet.of(
  "S3",
  "S3-CRT",
  "CloudFront KeyValueStore",
  "SESv2",
  "EventBridge",
)
```

intead of having to have a static block, imo avoiding the static block is nice

Reply:
updated, ty